docs/milestone-1-validation.md

```diff
@@ -0,0 +1,97 @@
+# Milestone-1 Validation
+
+* Scality Object Storage Interoperability Service (for short, **Scality OSIS**)
+* VMware Cloud Director Object Storage Extension (for short, **vCloud Director OSE**)
+
+## Prerequisites
+
+1. Run Vault & Cloud Server on a VM and expose the URL and Port
+
+1. Update environment file `application.properties` with Vault target URL (before starting Scality OSIS application)
+    * Vault URL should be set at property `osis.scality.vault.endpoint` (Refer Sample `application.properties` [here](../src/main/resources/application.properties))
+    * Vault Super admin credentials should be updated at properties `osis.scality.vault.access-key` and `osis.scality.vault.secret-key`
+
+1. Run the Scality OSIS application (Refer [here](../README.md#running-the-application))
+
+1. On OSE machine, vCloud Director OSE and OSE UI should be installed and running successfully.
```

Comment:
Reference to documentation will be helpful

Reply:
As the documentation is in a google docs document, I have not mentioned it. I can add it on https://scality.atlassian.net/browse/S3C-4253